src/components/Picker/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './Picker';
```

Comment:
There is no default export?

Reply:
i have  followed other components convention in which each component is only named exported as  this plugin is only for mobile not web